src/2_RTB/NormeDiProgetto/content.typ

```diff
@@ -414,3 +492,51 @@ I ruoli assunti sono i seguenti:
 === Cambio dei ruoli
 
 Per fare in modo che ogni membro svolga almeno 1 volta tutti i ruoli di cui sopra, il team si impegna a cambiarli ogni settimana.
+
+=== Procedure 
+
+==== Gestione delle comunicazioni
+
+===== Comunicazioni Interne 
+
+Riguardano esclusivamente i membri del Team e si svolgono tramite:
+
+        - *_Whatsapp_*: per una comunicazione veloce;
+        - *_Discord_*: piattaforma per comunicare che permette di:
+                + Creare server suddivisibili in vari canali testuali o vocali, dove verranno svolte le riunioni;
+                + Integrazioni con servizi esterni ad esempio Github.
+
+===== Comunicazioni Esterne
+
+Le comunicazioni esterne vengono affidate al _Responsabile_ attraverso i seguenti mezzi:
+
+        - _Email_ : si usa l'email di gruppo swateng.team\@gmail.com;
+        - _Element_: si usa il canale creato apposta dall'azienda Synclab per avere una comunicazione diretta.
+
+==== Gestione degli Incontri 
+
+===== Incontri Interni 
+
+Negli incontri interni possono partecipare solamente i membri del gruppo. Si svolgono principalmente una volta a settimana, il giorno può variare in caso di imprevisti, ma solitamente si tiene il venerdì mattina in modalità sincrona.
+
+Le linee guida per le riunioni:
+                + Prima dell'incontro avere un ordine del giorno, ovvero i punti eventuali da discutere;
+                + Discussione dei punti;
+                + Pianificazione attività per la settimana e assegnazione issue.
+                
+        Alla fine dell'incontro:
+                + Il responsabile ha il compito della stesura del verbale interno, fornendo una sintesi dei punti salienti dell'incontro. 
+
+Gli incontri hanno due modalità:
+
+        - *Fisici*: per gli stand-up meeting quotidiani (#glossary("Daily Scrum_")) di 5 minuti in cui si discutono brevemente le attività completate il giorno precedente e si espongono le attività pianificate per il futuro;
+        - *Virtuali*: si svolgono chiamate o video di gruppo in cui si discutono eventuali dubbi o difficoltà riscontrate. Lo strumento adatto per questo scopo è Discord.
+
+===== Incontri Esterni 
+
+Negli incontri esterni i partecipanti sono i membri del team e il/i referenti della Proponente. Questi incontri sono pianificati ogni due settimane in concomitanza con l'inizio e fine sprint. Durante queste sessioni, i partecipanti hanno l'opportunità di presentare gli sviluppi recenti, condividere i progressi raggiunti e discutere eventuali sfide o questioni emerse nel corso del lavoro. 
+Inoltre si possono richiedere incontri di formazione specifici su particolari tecnologie, questo offre al team l'opportunità di approfondire la comprensione di una tecnologia specifica, imparare le best practice e acquisire competenze più avanzate.
+
+Gli incontri sono prevalentemente in modalità sincrona.
```

Comment:
- il/i referenti della Proponente --> i referenti della Proponente
-  i partecipanti del team hanno l'opportunita'..
-  su particolari tecnologie, questo offre al team --> su particolari tecnologie, il che offre al team..
- _best practice_
- toglierei "Gli incontri sono prevalentemente in modalità sincrona."